x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/plan/physical/FieldExtractExecSerializationTests.java

```diff
@@ -18,7 +19,7 @@ public static FieldExtractExec randomFieldExtractExec(int depth) {
         Source source = randomSource();
         PhysicalPlan child = randomChild(depth);
         List<Attribute> attributesToExtract = randomFieldAttributes(1, 4, false);
-        return new FieldExtractExec(source, child, attributesToExtract);
+        return new FieldExtractExec(source, child, attributesToExtract, MappedFieldType.FieldExtractPreference.NONE);
```

Comment:
Could revert this change, if we use the pattern described in the comments on FieldExtractPreference